src/states/elder/mod.rs

```diff
@@ -1161,7 +1173,11 @@ impl Elder {
     // Constructs a signed message, finds the nodes responsible for accumulation, and either sends
     // these nodes a signature or tries to accumulate signatures for this message (on success, the
     // accumulator handles or forwards the message).
-    fn send_routing_message(&mut self, routing_msg: RoutingMessage) -> Result<(), RoutingError> {
```

Comment:
the `node_knowledge_override` is only populated when handle_candidate_approval.
maybe worth to put a comment here to explain the usage?